playbooks/deprovision.yaml

```diff
@@ -0,0 +1,30 @@
+---
+-
+  connection: local
+  gather_facts: false
+  hosts: localhost
+  name: "Playbook to deprovision Thoth's Nepthys deployment"
+  tags:
+    - openshift
+    - thoth
+    - nepthys
+
+  vars:
+    NEPTHYS_APPLICATION_NAMESPACE: "{{ lookup('env','NEPTHYS_APPLICATION_NAMESPACE') }}"
+    OPENSHIFT_AUTH_TOKEN: "{{ lookup('env','OCP_TOKEN') }}"
+    OPENSHIFT_MASTER_URL: "{{ lookup('env','OCP_URL') }}"
+
+  tasks:
+    - changed_when: false
+      command: "oc login {{ OCP_URL }} --insecure-skip-tls-verify=true --token {{ OCP_TOKEN }}"
+      name: "login to OpenShift using provided token"
+    - command: "oc delete {{ item }} --namespace {{ NEPTHYS_APPLICATION_NAMESPACE }}"
+      name: "deleting Objects"
+      with_items:
```

Comment:
I usually use label selector here for the whole app (`all=nepthys`), not to have it written verbatim in template (if there are some future additions). But that's just nitpicking... Feel free to keep it as is.

Reply:
I didn't the process of removing pod/job/build/etc through app tag. will look into it and update this.
great advise, thanks :)